lib/new_relic/agent/instrumentation/action_dispatch_subscriber.rb

```diff
@@ -0,0 +1,64 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/agent/instrumentation/notifications_subscriber'
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      class ActionDispatchSubscriber < NotificationsSubscriber
+        def start(name, id, payload)
```

Comment:
since my branch isn't done yet, after this PR is merged i can pull the dev updates into my branch and update this section to use the new shared version of these methods, if that sounds good to you.

Reply:
Certainly, feel free!